src/Assignment.sql

```diff
@@ -0,0 +1,1259 @@
+            | 1002 | Serres  |   13 | SanJose   |
+            | 1003 | AxelRod |   10 | New York  |
+            | 1004 | Motika  |   11 | London    |
+            | 1007 | Rifkin  |   15 | Barcelona |
+            | 1008 | Fran    |   25 | London    |
+            +------+---------+------+-----------+
+------------------------------------------------------------------------------------------------------------------------
+32) Select all customers whose names fall in between ‘A’ and ‘G’ alphabetical range.
+Query :-
+        select CNAME from Customers where CNAME between 'A' and 'G';
+Output :-
+           +---------+
+           | CNAME   |
+           +---------+
+           | Clemens |
+           | Cineros |
+           +---------+
+------------------------------------------------------------------------------------------------------------------------
+33) Select all the possible combinations of customers you can assign.
+Query :-  select c.CNAME,s.SNAME from Customers c join  Salespeople s on c.SNUM=s.SNUM;
+Output :- +----------+---------+
+          | CNAME    | SNAME   |
+          +----------+---------+
+          | Hoffman  | Peel    |
+          | Giovanni | AxelRod |
+          | Liu      | Serres  |
+          | Grass    | Serres  |
+          | Clemens  | Peel    |
+          | Pereira  | Motika  |
+          | Cineros  | Rifkin  |
+          +----------+---------+
+------------------------------------------------------------------------------------------------------------------------
+34) Select all orders that are greater than the average for October 4.
+Query :-  SELECT * FROM Orders WHERE AMT>(SELECT  AVG(AMT) FROM Orders WHERE ODATE ='1990-04-10');
+Output :-  +------+---------+------------+------+------+
+           | ONUM | AMT     | ODATE      | CNUM | SNUM |
+           +------+---------+------------+------+------+
+           | 3002 |  1900.1 | 1990-03-10 | 2007 | 1004 |
+           | 3005 | 5160.45 | 1990-03-10 | 2003 | 1002 |
+           | 3006 | 1098.16 | 1990-03-10 | 2008 | 1007 |
+           | 3008 |    4723 | 1990-05-10 | 2006 | 1001 |
+           | 3009 | 1713.23 | 1990-04-10 | 2002 | 1003 |
+           | 3010 | 1309.95 | 1990-06-10 | 2004 | 1002 |
+           | 3011 | 9891.88 | 1990-06-10 | 2006 | 1001 |
+           +------+---------+------------+------+------+
+------------------------------------------------------------------------------------------------------------------------
+35)   Write a select command using correlated subquery that selects the
+      names and numbers of all customers with ratings equal to the maximum for their city.
+------------------------------------------------------------------------------------------------------------------------
+36) Write a query that totals the orders for each day and places the results in descending order.
+Query :-    select COUNT(ODATE) "TOTAL ORDERS", ODATE  FROM Orders group by ODATE order by "TOTAL ORDERS";
```

Comment:
Here the sum of the amount will be done on each date instead of order count. This is a wrong query.